src/test/java/fi/metatavu/metaform/server/test/functional/builder/impl/ReplyTestBuilderResource.kt

```diff
@@ -119,9 +120,11 @@ class ReplyTestBuilderResource(
             includeRevisions: Boolean?,
             fields: Array<String>?,
             firstResult: Int?,
-            maxResults: Int?
+            maxResults: Int?,
+            orderBy: ReplyOrderCriteria?,
+            latestFirst: Boolean?
     ): Array<Reply> {
-        return api.listReplies(metaformId, userId, createdBefore, createdAfter, modifiedBefore, modifiedAfter, includeRevisions, fields, firstResult, maxResults)
+        return api.listReplies(metaformId, userId, createdBefore, createdAfter, modifiedBefore, modifiedAfter, includeRevisions, fields, firstResult, maxResults, orderBy, latestFirst)
```

Comment:
In Kotlin named syntax is usually preferred way to pass variables to methods that have multiple parameters. This is just a reminder and no need to do anything about it in this case since the whole project is done without named syntax